src/ports/tabby/index.ts

```diff
@@ -0,0 +1,27 @@
+import { AuraAPI } from 'core'
+import { resolve } from 'path'
+
+export async function TabbyPort(Aura: AuraAPI) {
+  const { createPort, createReadme, colorSchemes } = Aura
+
+  const portName = 'Tabby'
+  const version = '1.0.0'
+  const displayName = 'Aura Theme'
```

Comment:
This value should come from Aura constants info, I sent you in the review the full content of this file to override it.